test/common/tests.js

```diff
@@ -1467,6 +1467,53 @@ module.exports = (sql, driver) => {
       })
     },
 
+    'streaming trailing rows' (done) {
+      let rows = 0
+      let started = false
+
+      const req = new TestRequest()
+      req.stream = true
+      req.pause()
+      req.query('select top 102 * from streaming')
+      req.on('error', (err) => {
+        if (err.code !== 'ECANCEL') {
+          req.cancel()
+          clearTimeout(timeout)
+          done(err)
+        }
+      })
+
+      // start the request after 1 second
+      const timeout = setTimeout(() => {
+        assert.ok(!started)
+        assert.strictEqual(rows, 0)
+        started = true
+        req.resume()
+      }, 1000)
```

Comment:
waiting a second in the test suite to run this test is quite a long time. I'm not convinced we need to delay the start of the stream because this functionality is tested in other tests.